.github/workflows/linter.yml

```diff
@@ -0,0 +1,55 @@
+---
+#################################
+#################################
+## Super Linter GitHub Actions ##
+#################################
+#################################
+name: Lint Code Base
+
+#
+# Documentation:
+# https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore: [dev]
+    # Remove the line above to run when pushing to master
```

Comment:
Not a big deal but maybe we should change that comment to say 'main' for consistency. Mostly in case someone new to the code looks here in the future.